src/unix/bsd/netbsdlike/mod.rs

```diff
@@ -837,6 +837,19 @@ extern "C" {
         fd: ::c_int,
         newfd: ::c_int,
     ) -> ::c_int;
+    pub fn sendmmsg(
+        sockfd: ::c_int,
+        msgvec: *mut ::mmsghdr,
+        vlen: ::c_uint,
+        flags: ::c_int,
+    ) -> ::c_int;
+    pub fn recvmmsg(
+        sockfd: ::c_int,
+        msgvec: *mut ::mmsghdr,
```

Comment:
`msgvec` is `mmsghdr` per https://man.openbsd.org/sendmsg.2 and https://man.openbsd.org/recvmsg.2